src/activity_handlers.cpp

```diff
@@ -890,7 +890,7 @@ void activity_handlers::firstaid_finish( player_activity *act, player *p )
 static void rod_fish( player *p, int sSkillLevel, int fishChance )
 {
    if( sSkillLevel > fishChance ) {
-        std::vector<monster *> fishables = g->get_fishable(60); //get the nearby fish list.
+        std::vector<monster *> fishables = g->get_fishable(DISTANCE_FROM_CENTER); //get the nearby fish list.
```

Comment:
This is range at which you can attract fish to a lure, 60 already seems a bit overboard...

Reply:
Renamed to fishing_distance.